docs/index.md

```diff
@@ -32,32 +34,28 @@ The code is kept here for easy maintenance.
 </p>
 <h3 align="center">Code scanning at ludicrous speed.<br />Find bugs and reachable dependency vulnerabilities in code.<br />Enforce your code standards on every commit.</h3>
 
-Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting dependency vulnerabilities, and enforcing code standards. [Get started →](/getting-started/quickstart)
-
-Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**.
-
-Its rules look like the code you already write; no abstract syntax trees, regex wrestling, or painful DSLs. Here's a rule for finding Python `print()` statements. Run it by clicking the [▸] button:
```

Comment:
I’m curious about the reasoning to not mention ASTs, regex, DSLs. Can you share more about that?

Reply:
I moved this up:

<img width="881" alt="image" src="https://github.com/semgrep/semgrep-docs/assets/16521651/26b57aba-aaab-48cc-bf64-e01ffeb7139b">